webapp/components/create_comment.jsx

```diff
@@ -123,6 +148,41 @@ export default class CreateComment extends React.Component {
 
         MessageHistoryStore.storeMessageInHistory(this.state.message);
 
+        this.setState({submitting: true, serverError: null});
+
+        if (post.message.indexOf('/') === 0) {
+            PostStore.storeCommentDraft(this.props.rootId, null);
+            this.setState({message: '', postError: null, fileInfos: []});
+
+            post.channel_id = this.props.channelId;
+            post.root_id = this.props.rootId;
+            post.parent_id = this.props.rootId;
+            ChannelActions.executeCommand(
+                post,
+                (data) => {
+                    this.setState({submitting: false});
+
+                    if (data.goto_location && data.goto_location.length > 0) {
+                        browserHistory.push(data.goto_location);
```

Comment:
In my local when `goto_location` points to `http://www.google.com`, my browser show Mattermost select team page.
Should I fix this? or wait new issue?